packages/react/src/date-picker/Calendar.tsx

```diff
@@ -438,6 +438,7 @@ function YearMonthSelect({
 			</label>
 			<Box
 				as="select"
+				autoComplete="off"
```

Comment:
Feels silly, but it appeases ARC and I guess prevents autofill from accidentally populating the date picker dropdowns